__init__.py

```diff
@@ -199,6 +199,28 @@ def _handle_switch(self, message):
         LOGGER.debug("Entity: %s" % entity)
         LOGGER.debug("Action: %s" % action)
 
```

Comment:
There was a separate logger PR but it appears this got missed

Reply:
Updated with upstream